src/core/MRApp.js

```diff
@@ -113,7 +114,15 @@ export class MRApp extends MRElement {
         this.observer = new MutationObserver(this.mutationCallback);
         this.observer.observe(this, { attributes: true, childList: true });
 
+        if (this.getAttribute('stats')) {
+            const statsEl = document.createElement('mr-stats');
+            statsEl.setAttribute('data-comp-anchor', 'type: plane; label: wall;');
+            statsEl.setAttribute('data-position', '0 0.2 0.01');
+            this.appendChild(statsEl);
+        }
+
```

Comment:
this would be better off in the constructor of `StatsSystem` as we should only have a stats system if `this.getAttribute('stats')` is true

Reply:
Not a final conclusion yet but I'm thinking of also allowing users to write `<mr-stats>` explicitly in their contents, like the following, so initializing it outside of the system constructor for now.

```
<mr-stats data-comp-anchor="..." " data-position="..."/>
```

Reply:
i think that's a good option since it aligns much more with the idea of how we're using it as an entity

we can also allow for both - kind of like how MRSkybox you can set it as `<mr-skybox...>` or just directly through the css of the `mr-app`'s background property

Reply:
also - something that might be helpful as your implementation will potentially be creating <mr-stats> on the fly as well
- see the note i had mentioned about MRHyperlink (since it makes its dom item directly in the entity)
- and then i forgot to link this for MRSkybox: https://github.com/Volumetrics-io/mrjs/blob/main/src/core/MRApp.js#L248 <-- this is a good reference for adding in elements on the fly into the MRApp s.t. it works nicely with our pre-existing system